cob_gazebo_worlds/launch/world.launch

```diff
@@ -34,6 +37,8 @@
 	</include>
 
 	<!-- spawn environment and start environment-specific nodes -->
-	<include file="$(arg pkg_gazebo_worlds)/launch/$(arg robot_env).xml"/>
+	<include file="$(arg pkg_gazebo_worlds)/launch/$(arg robot_env).xml">
+		<arg name="pkg_gazebo_worlds" value="$(arg pkg_gazebo_worlds)"/>
+	</include>
```

Comment:
please revert these lines and add a `launch/$(arg robot_env).xml` to your external `pkg_env_config` package

then you do not need to pass `pkg_gazebo_worlds` to all the other `*.xml` files in this package